examples/kitchensink/app-counter/package.json

```diff
@@ -4,9 +4,34 @@
   "description": "Kitchensink counter app in Frint",
   "main": "index.js",
   "scripts": {
-    "build": "../../../node_modules/.bin/webpack"
+    "build": "./node_modules/.bin/webpack",
```

Comment:
local bins don't need full path with `node_modules`, just `webpack ...` is enough.